src/resources/views/inc/show_fields.blade.php

```diff
@@ -1,9 +1,14 @@
 {{-- Show the inputs --}}
 @foreach ($fields as $field)
-    <!-- load the view from the application if it exists, otherwise load the one in the package -->
-    @if(view()->exists('vendor.backpack.crud.fields.'.$field['type']))
-        @include('vendor.backpack.crud.fields.'.$field['type'], array('field' => $field))
+    <!-- load the view from the view attribute if set -->
+    @if(isset($field['view']))
+        @include($field['view'].'.'.$field['type'], ['field' => $field])
```

Comment:
Here I would expect the `view`attribute contains the whole view path and will not be appended by `type`. See my comment at #1385